solr/modules/hdfs/README.md

```diff
@@ -0,0 +1,31 @@
+Apache Solr HDFS Module
+===============================
+
+Introduction
+------------
+This module implements the support for Hadoop Distributed File System in Apache Solr. 
+
+Building
+--------
+The HDFS module uses the same Gradle build as the core Solr components. 
+
+To build the module, you can use
+
+```
+./gradlew :solr:modules:hdfs:assemble
+```
+
+The resulting module will be placed to the libs directory, for example:
+`solr/modules/hdfs/build/libs/solr-hdfs-9.0.0-SNAPSHOT.jar`
+
+To execute the module tests:
+
+```
+./gradlew :solr:modules:hdfs:test
+```
+
+Usage
+-----
+To use the module, it needs to be placed to the Solr web application classpath (for example by symlinking it to the WEB-INF/lib directory.)
```

Comment:
IMO, unless there are unique constraints about HDFS's JARs, we can simply refer to the ref guide (`solr-plugins.adoc`)  on how plugins are installed.  There are a number of options.

Reply:
Addressed in bb216a74511e8a13787556ff94da35332ca71692